pkg/server/test/list_objects.go

```diff
@@ -200,11 +194,9 @@ func TestListObjectsRespectsMaxResults(t *testing.T, ds storage.OpenFGADatastore
 				ListObjectsMaxResults: test.maxResults,
 				ConnectedObjects:      connectedObjCmd.StreamedConnectedObjects,
 				ResolveNodeLimit:      defaultResolveNodeLimit,
-				CheckResolver:         checker,
 			}
 			typesys := typesystem.New(model)
 			ctx = typesystem.ContextWithTypesystem(ctx, typesys)
-			ctx = storage.ContextWithContextualTuples(ctx, test.contextualTuples.GetTupleKeys())
```

Comment:
this is my ultimate goal, get rid of this line. I should not have to remember to set the `ctx` when writing a test